app/controllers/pbs/events_controller.rb

```diff
@@ -72,6 +73,22 @@ def assign_contact_attrs_with_supercamp_flag
     contact_attrs
   end
 
+  def preview_camp_application_validation
+    return unless entry.is_a?(Event::Campy)
+    return unless [entry.leader, entry.coach].include? current_person
+    return if entry.camp_submitted?
+
+    original_camp_submitted_at = entry.camp_submitted_at
+    entry.camp_submitted_at = Time.zone.now.to_date
```

Comment:
Zweitteilige Suggestion: Man kann AR nutzen, um das Attribute wieder zurückzusetzen. (1/2)

Also muss es hier nicht gespeichert werden:

```suggestion
    entry.camp_submitted_at = Time.zone.now.to_date
```